### ISSUE TITLE
Disabling correlation globaly (from site admin)

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#987]
Disabling correlation (from site admin) would be nice feature. If you want to bootstrap a MISP with a lot of events like importing a huge set of already processed malware, it could be slow as the correlation is done for each import.

**Comment by adulau** [comment]
In 2.4.58. Fixed

**Comment by anaike** [comment]
Hello,

I set up a MISP to MISP connection between one empty MISP instance (A) and another one that has more than 300k events (B). In one month the number of events that the MISP A fetched from MISP B is about 100k. To try to speed up the fetching process (and also because i saw that the size of the correlation table was growing exponentially), I disabled the correlation feature enabling the _MISP.completely_disable_correlation_ setting. After disabling the correlation, a job was run and it deleted all data from the correlations table, whose size had reached 0 MB. Then i restarted the fetching process but it is, however, very slow. I saw that after a couple of weeks, the correlations table returned to having the size of 146777.84MB. 

Is this normal behavior? I would have expected correlation to be disabled for new events as well.

The "affected" MISP version is 2.4.118.
